inc/Services/Facet_WP.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace BEA\Theme\Framework\Services;
+
+use BEA\Theme\Framework\Service;
+use BEA\Theme\Framework\Service_Container;
+
+class Facet_WP implements Service {
+	public function register( Service_Container $container ): void {
+	}
+
+	public function boot( Service_Container $container ): void {
+		add_filter( 'facetwp_load_assets', '__return_true' );
```

Comment:
@MarieComet a commenter par défaut

Reply:
il manque 
`add_filter( 'facetwp_facet_html', [ $this, 'accessible_facetwp_labels' ], 10, 2 );` 

``` /** Fix Labels for supported facets.
	 *  Put in show_label_not_empty the facets that only need to be visible in they have results
	 *
	 * @param string $html
	 * @param array $args
	 *
	 * @return string
	 */
	public function accessible_facetwp_labels( string $html, array $args ): string {
		$show_label_not_empty = [
			'checkboxes',
			'radio',
			'reset',
		];

		if ( ( true === in_array( $args['facet']['type'], $show_label_not_empty, true ) && ! empty( $args['values'] ) ) || false === in_array( $args['facet']['type'], $show_label_not_empty, true ) ) {
			$label = $args['facet']['label'];
			if ( function_exists( 'facetwp_i18n' ) ) {
				$label = facetwp_i18n( $label );
			}

			// Add id attribute for dropdown facets
			if ( 'dropdown' === $args['facet']['type'] ) {
				$html = sprintf( '<label class="facetwp-label" for="%s">%s</label>%s', esc_attr( $args['facet']['name'] ), esc_html( $label ), $html );
				$html = str_replace( '<select class=', '<select id="' . esc_attr( $args['facet']['name'] ) . '" class=', $html );
			}

			if ( 'search' === $args['facet']['type'] ) {
				$html = sprintf( '<label class="facetwp-label" for="facetwp-%s">%s</label>%s', esc_attr( $args['facet']['name'] ), esc_html( $label ), $html );
				$html = str_replace( '<input type=', '<input id="facetwp-' . esc_attr( $args['facet']['name'] ) . '" type=', $html );
			}
		}

		return $html;
	}```